examples/README.md

```diff
@@ -68,7 +68,7 @@ Supports multiple lines where the export is just somewhere in between:
 
 ```sh
 echo "Auth token for service foo"
-export SERVICE_FOO_TOKEN=foobar
```

Comment:
Setting the env var without quotes was intentional so we can check if Runme can handle both, is this not supported anymore?